src/target/emu_devo7e/Makefile.inc

```diff
@@ -2,7 +2,7 @@ SCREENSIZE  := 128x64x1
 FILESYSTEMS := common base_fonts 128x64x1
 FONTS        = filesystem/$(FILESYSTEM)/media/12normal.fon \
                filesystem/$(FILESYSTEM)/media/04b03.fon
-
+LANGUAGE = devo7e
```

Comment:
Display size is the same for devo7e and devo10, so no need for another string alternative